docs/docs/concepts/05-building-blocks/01-signals.md

```diff
@@ -4,7 +4,7 @@ sidebar_position: 1
 
 # Signals
 
-In the AICA framework, signals are designed to exchange continuous data between components and controllers in a running
+In the AICA System, signals are designed to exchange continuous data between components and controllers in a running
```

Comment:
```suggestion
In AICA System, signals are designed to exchange continuous data between components and controllers in a running
```
to be consistent

Reply:
actually while AICA Studio and AICA Core are products and don't need the definite article, "AICA System" is less tangible in my opinion. The way I look at it is that "AICA System" is equivalent to "AICA (software) ecosystem" - and I would always say "the ecosystem". 

However something like AICA Universe goes so big and all-encompassing that it's more of a brand than anything else, like a theme park that all of AICA products, partners, users and services fit inside.

What works without "the":
- AICA Studio
- AICA Core
- AICA Universe

What doesn't work without "the":
- AICA System